e2e/tests/nightly/admin/institution-admin-ui.spec.ts

```diff
@@ -0,0 +1,87 @@
+    await institutionEditPage.waitForLoad();
+
+    // verify the input fields
+    const institutionName = await institutionEditPage.getInstitutionNameValue();
+    expect(institutionName).toBe(testInstitutionName);
+    expect(await institutionEditPage.getInstitutionTypeValue()).toBe(InstitutionTypeSelectValue.Other);
+    expect(await institutionEditPage.getRTeRAtoggle().isChecked()).toBe(true);
+    expect(await institutionEditPage.getRTEmailAcceptedValue()).toBe(AcceptedAddressSelectValue.Individual);
+
+    //verify that the Accepted Email Addresses text area is present in registered-card-details
+    institutionEditPage.getRTEmailAddressInput();
+    expect(await institutionEditPage.getCTEnabledtoggle().isChecked()).toBe(true);
+    expect(await institutionEditPage.getCTEmailAcceptedValue()).toBe(AcceptedAddressSelectValue.Domains);
+    // verify that the Accepted Email Domains text area is present in controlled-card-details
+    institutionEditPage.getCTEmailDomainsInput();
+    //verify that the save button is disabled
+    expect(await institutionEditPage.getSaveButton().isCursorNotAllowed()).toBe(true);
+    await institutionEditPage.clickCancelButton();
+    await institutionAdminPage.waitForLoad();
+  });
+
+  test('add new Institution page UI check', async () => {
+    const institutionAdminPage = new InstitutionAdminPage(page);
+    await institutionAdminPage.waitForLoad();
+    await institutionAdminPage.getCreateNewInstitutionBtn().click();
+    const institutionEditPage = new InstitutionEditPage(page);
+    await institutionEditPage.waitForLoad();
+    await institutionEditPage.getAddNewInstitutionFields();
+    expect(await institutionEditPage.getRTeRAtoggle().isChecked()).toBe(true);
+    await institutionEditPage.selectRTEmailOption(AcceptedAddressSelectValue.Domains);
+    institutionEditPage.getRTEmailDomainsInput();
+    expect(await institutionEditPage.getCTeRAtoggle().isDisabled()).toBe(true);
+    expect(await institutionEditPage.getCTEnabledtoggle().isChecked()).toBe(false);
+    await institutionEditPage.clickCTEnabledtoggle();
+    expect(await institutionEditPage.getCTEnabledtoggle().isChecked()).toBe(true);
+    expect(await institutionEditPage.getCTEmailAcceptedValue()).toBe(AcceptedAddressSelectValue.Domains);
+
+    //verify if the Accepted Email Domains textarea now displays in controlled-card-details div
+    institutionEditPage.getCTEmailDomainsInput();
+    // select from dropdown option- Individual email address is listed below:
+    await institutionEditPage.selectCTEmailOption(AcceptedAddressSelectValue.Individual);
+    //verify if the Accepted Email Addresses textarea now displays in controlled-card-details div
+    institutionEditPage.getCTEmailAddressInput();
+
+    // verify that the ADD button is disabled
+    expect(await institutionEditPage.getAddButton().isCursorNotAllowed()).toBe(true);
+    await institutionEditPage.clickCancelButton();
+    await institutionAdminPage.waitForLoad();
```

Comment:
@jmthibault79 @aweng98 this test fails at line 85, whereas it passes for the same test step at line [56](https://github.com/all-of-us/workbench/pull/5615/files#diff-6d42b8e08a4934274fbc70508ca5341522626cb4006ac0e9532df6bb950826f3R56) sharing the stacktrace here. Could you please look into it I'm unable to figure out what is the issue. Thanks!

```
● Institution Admin › add new Institution page UI check

    Protocol error (Runtime.callFunctionOn): Target closed.

      at node_modules/puppeteer/lib/cjs/puppeteer/common/Connection.js:208:63
      at CDPSession.send (node_modules/puppeteer/lib/cjs/puppeteer/common/Connection.js:207:16)
      at ExecutionContext._evaluateInternal (node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:200:50)
      at ExecutionContext.evaluateHandle (node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:151:21)
      at WaitTask.rerun (node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:528:37)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 skipped, 2 total
Snapshots:   0 total
Time:        40.09 s
Ran all test suites matching /tests\/nightly\/admin\/institution-admin-ui.spec.ts/i.
**  jest-stare --reporters: wrote output report to logs/puppeteer-test-results.html	**
```

Reply:
OK, that is surprising.  In manual testing, Cancel seems to work fine in all cases.  I'll take a look.

Reply:
test passed for me locally.  I don't know what's going on here.